python/lsst/ts/Dome/dome_csc.py

```diff
@@ -506,10 +519,33 @@ def remove_keys_from_dict(self, dict_with_too_many_keys, keys_to_remove):
         dict_with_keys_removed = {
             x: dict_with_too_many_keys[x]
             for x in dict_with_too_many_keys
-            if x not in keys_to_remove
+            if x not in _KEYS_TO_REMOVE
         }
         return dict_with_keys_removed
 
+    # noinspection PyMethodMayBeStatic
+    def convert_telemetry_radians_to_degrees(self, telemetry_in_radians):
+        telemetry_in_degrees = {}
+        for key in telemetry_in_radians.keys():
+            if key in _KEYS_IN_RADIANS:
+                telemetry_in_degrees[key] = (
+                    telemetry_in_radians[key] * _RADIANS_TO_DEGREES
+                )
+            else:
+                telemetry_in_degrees[key] = telemetry_in_radians[key]
```

Comment:
Nice way to handle this. The "else" caught me off guard at first (setting degrees to radians?), but I have no idea how to make it less startling, except perhaps a code comment such as "not an angle".

Reply:
Done.